src/main/java/de/retest/recheck/ui/descriptors/Element.java

```diff
@@ -104,6 +104,14 @@ public Element applyChanges( final ActionChangeSet actionChangeSet ) {
 		return element;
 	}
 
+	/**
+	 * Note that the retestId is immutable, so this method returns a new Element that is a copy of the old one with the
+	 * new retestId.
+	 */
+	public Element setRetestId( final String retestId ) {
```

Comment:
I wouldn't call the method `set` because it don't work like a typical `set*` method. I prefer something like `applyRetestId`.